pkgs/development/coq-modules/jasmin-proofs/default.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  mkCoqDerivation,
+  coq,
+  mathcomp,
+  mathcomp-word,
+  version ? null,
+}:
+
+mkCoqDerivation {
+  pname = "jasmin-proofs";
+  owner = "jasmin-lang";
+  repo = "jasmin";
+
+  inherit version;
+  defaultVersion =
+    with lib.versions;
+    lib.switch [ coq.version mathcomp.version ] [
+      { cases = [ (range "8.20" "8.20") (range "2.3" "2.3") ];
+        out = "2024.07.2"; }
+    ] null;
+  releaseRev = v: "v${v}";
+
+  release."2024.07.2".sha256 = "sha256-aF8SYY5jRxQ6iEr7t6mRN3BEmIDhJ53PGhuZiJGB+i8=";
+
+  propagatedBuildInputs = [
+    mathcomp-word
+  ];
+
+  makeFlags = [ "-C" "proofs" ];
+
+  meta = with lib; {
+    description = "Proofs for Jasmin compiler";
```

```suggestion
    description = "Jasmin language & verified compiler";
```

or something like that…